### PR TITLE
Close #3542. Documentation fix for create_network_block

### LINKED ISSUE #3542: Icepak.create_network_blocks(input_list, gravity_dir=2, ...) creates networks with wrong board and case sides
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

The function `Icepak.create_network_blocks(input_list, gravity_dir=2, ...)` creates thermal networks with **wrong** board and case sides. As I understand it, the parameter `gravity_dir` is actually the direction of case side of the component.  `gravity_dir=2` should mean  the case is in the Z- direction of the component, while `gravity_dir=5` means the case is in the Z+ direction. However, when I call this function with `gravity_dir=2`, the generated networks have the Rjc and the top face (Z+) of the component on the same side, which is the same as when `gravity_dir=5`. 

Furthermore, the [example](https://aedt.docs.pyansys.com/version/stable/API/_autosummary/pyaedt.icepak.Icepak.create_network_blocks.html) in the document calls an unavailable attribute `icepak.GravityDirection.ZNeg`. 

### Steps To Reproduce

```python
from pyaedt import Icepak

ipk = Icepak(new_desktop_session=True)

box1 = ipk.modeler.create_box([1, 2, 1], [6, 5, 3], "NetworkBox2", "copper")
box2 = ipk.modeler.create_box([8, 9, 1], [5, 5, 4], "NetworkBox3", "copper")
network_list = [["NetworkBox2", 20, 10, 3], ["NetworkBox3", 4, 10, 2]]
blocks = ipk.create_network_blocks(network_list, 5, 1, False)

ipk.release_desktop(close_projects=False, close_desktop=False)
``` 

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.7

### Installed packages

attrs==23.1.0
backcall==0.2.0
certifi==2023.5.7
cffi==1.15.1
charset-normalizer==3.2.0
click==8.1.5
click-plugins==1.1.1
cligj==0.7.2
clr-loader==0.2.5
colorama==0.4.6
cycler==0.11.0
debugpy==1.6.7
decorator==5.1.1
entrypoints==0.4
et-xmlfile==1.1.0
Fiona==1.9.4.post1
fonttools==4.38.0
geopandas==0.10.2
idna==3.4
imageio==2.31.1
importlib-metadata==6.7.0
ipykernel==6.16.2
ipython==7.34.0
jedi==0.18.2
jupyter_client==7.4.9
jupyter_core==4.12.0
kiwisolver==1.4.4
matplotlib==3.5.3
matplotlib-inline==0.1.6
nest-asyncio==1.5.6
networkx==2.6.3
numpy==1.21.6
openpyxl==3.1.2
osmnx==1.1.2
packaging==23.1
pandas==1.3.5
parso==0.8.3
pickleshare==0.7.5
Pillow==9.5.0
platformdirs==3.9.1
plumbum==1.8.2
pooch==1.7.0
prompt-toolkit==3.0.39
psutil==5.9.5
pyaedt==0.6.87
pycparser==2.21
Pygments==2.15.1
pyparsing==3.1.0
pyproj==3.2.1
python-dateutil==2.8.2
pythonnet==3.0.1
pytz==2023.3
pyvista==0.38.0
pywin32==306
pyzmq==25.1.0
requests==2.31.0
rpyc==5.3.1
Rtree==1.0.1
scikit-rf==0.28.0
scipy==1.7.3
scooby==0.7.2
Shapely==1.8.5.post1
six==1.16.0
SRTM.py==0.3.7
tornado==6.2
traitlets==5.9.0
typing_extensions==4.7.1
urllib3==2.0.3
utm==0.7.0
vtk==9.2.6
wcwidth==0.2.6
zipp==3.15.0

### PR DESCRIPTION
**Pull request by sivasu37**
Close #3542. Documentation fix for create_network_block

### CODE REVIEW

File: pyaedt/icepak.py [hunk 1]
```diff
@@ -758,14 +758,16 @@ def create_network_block(
...
+        rjb : float
             RJB value.
-        gravity_dir :
-            Gravity direction from -X to +Z. Options are ``0`` to ``5``.
-        top :
-            Board bounding value in millimeters of the top face.
+        gravity_dir : int
+            Gravity direction X to Z. Options are ``0`` to ``2``. Determines the orientation of network boundary faces
+        top : float, optional
+            Chosen orientation (X to Z) coordinate value in millimeters of the top face of board.
```
Comment (PipKat) [inline pyaedt/icepak.py:768]:
```suggestion
            Chosen orientation (X to Z) coordinate value in millimeters of the top face of the board.
```

File: pyaedt/icepak.py [hunk 2]
```diff
@@ -758,14 +758,16 @@ def create_network_block(
...
+            Determines the casing and board side of network.
+            The default is ''0 mm''.
```
Comment (PipKat) [inline pyaedt/icepak.py:770]:
```suggestion
```
The default should be after the brief description, with more information following.

File: pyaedt/icepak.py [hunk 3]
```diff
@@ -859,9 +861,11 @@ def create_network_blocks(
             List of sources with inputs ``rjc``, ``rjb``, and ``power``.
             For example, ``[[Objname1, rjc, rjb, power1, power2, ...], [Objname2, rjc2, rbj2, power1, power2, ...]]``.
         gravity_dir : int
-            Gravity direction from -X to +Z. Options are ``0`` to ``5``.
-        top :
-            Board bounding value in millimeters of the top face.
+            Gravity direction X to Z. Options are ``0`` to ``2``. Determines the orientation of network boundary faces.
```
Comment (PipKat) [inline pyaedt/icepak.py:864]:
```suggestion
            Gravity direction X to Z. Options are ``0`` to ``2``. This parameter determines the orientation of network boundary faces.
```

File: pyaedt/icepak.py [hunk 4]
```diff
@@ -859,9 +861,11 @@ def create_network_blocks(
...
+        top : float, optional
+            Chosen orientation (X to Z) coordinate value in millimeters of the top face of board.
+            Determines the casing and board side of network.
+            The default is ''0 mm''.
```
Comment (PipKat) [inline pyaedt/icepak.py:868]:
```suggestion
            Chosen orientation (X to Z) coordinate value in millimeters of the top face of
            the board. The default is ''0 mm''. This parameter determines the casing and
            board side of the network.
            
```

**Review by lorenzovecchietti — APPROVED** [review]

File: pyaedt/icepak.py [hunk 5]
```diff
@@ -758,14 +758,16 @@ def create_network_block(
...
             RJC value.
-        rjb :
+        rjb : float
             RJB value.
-        gravity_dir :
-            Gravity direction from -X to +Z. Options are ``0`` to ``5``.
-        top :
-            Board bounding value in millimeters of the top face.
+        gravity_dir : int
+            Gravity direction X to Z. Options are ``0`` to ``2``. Determines the orientation of network boundary faces
```
Comment (MaxJPRey) [inline pyaedt/icepak.py:766]:
```suggestion
            Gravity direction X to Z. Options are ``0`` to ``2``. Determines the orientation of network boundary faces.
```

**Pull request merged**